### PR TITLE
Rewrite the verification system

### PR DESCRIPTION
**Pull request by Infiniticity**

### CODE REVIEW

**Review by Sachin-dot-py — APPROVED** [review]

File: app.py [hunk 1]
```diff
@@ -63,36 +63,30 @@ async def on_voice_state_update(member, before, after):
                 await before.channel.edit(name="General")  # Reset channel name
 
 
+session_roles = {782273117321166888, 782423045578948618, 782423808271056917, 853135207862370324, 853135509488009216, 853135836597583902, 931662314074169364, 921725505051459664, 921725841677901905, 698285991110246441}
```
Comment (vsmart-06) [inline app.py:66]: These 2 variables have to be changed to lists

Comment (vsmart-06) [inline app.py:66]: These 2 variables have to be changed to lists

File: app.py [hunk 2]
```diff
@@ -63,36 +63,30 @@ async def on_voice_state_update(member, before, after):
...
+
 @bot.event
 async def on_raw_reaction_add(reaction):
     guild = bot.get_guild(GUILD_ID)
     user = await guild.fetch_member(reaction.user_id)
     if user.bot:
         return
     is_rr = rrDB.get_rr(str(reaction.emoji), reaction.message_id)
+    verified = discord.utils.get(guild.roles, name="Verified")
```
Comment (vsmart-06) [inline app.py:76]: Use the utility function `getRole` over here

**Review by vsmart-06 — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (Infiniticity) [reply app.py:66]: No, they need to be sets for `intersection` to work.

Comment (Sachin-dot-py) [reply app.py:66]: I think a set was defined here, which is fine.

File: app.py (hunk 2 above)
Comment (Infiniticity) [reply app.py]: Done.

File: app.py (hunk 1 above)
Comment (vsmart-06) [reply app.py:66]: Oh ok nevermind then

**Review by vsmart-06 — APPROVED** [review]

**Review by Sachin-dot-py — CHANGES_REQUESTED** [review]
```
verified = getRole("Verified)
stage1 = getRole("Stage 1 - Unverified")
stage2 = getRole("Stage 2 - Unverified")
```
`getRole` needs to be `await`-ed
This change needs to be made in two places.

File: app.py [hunk 3]
```diff
@@ -199,9 +193,20 @@ async def on_raw_reaction_remove(reaction):
     if user.bot:
         return
     is_rr = rrDB.get_rr(str(reaction.emoji), reaction.message_id)
+    verified = getRole("Verified)
```
Comment (vsmart-06) [inline app.py:196]:
```suggestion
    verified = getRole("Verified")
```

**Review by vsmart-06 — CHANGES_REQUESTED** [review]

File: app.py [hunk 4]
```diff
@@ -63,36 +63,30 @@ async def on_voice_state_update(member, before, after):
...
+
 @bot.event
 async def on_raw_reaction_add(reaction):
     guild = bot.get_guild(GUILD_ID)
     user = await guild.fetch_member(reaction.user_id)
     if user.bot:
         return
     is_rr = rrDB.get_rr(str(reaction.emoji), reaction.message_id)
+    verified = await getRole("Verified)
```
Comment (vsmart-06) [inline app.py:76]:
```suggestion
    verified = await getRole("Verified")
```

**Review by vsmart-06 — CHANGES_REQUESTED** [review]

**Pull request merged**